clang/lib/AST/Expr.cpp

```diff
@@ -612,7 +600,12 @@ std::string SYCLUniqueStableIdExpr::ComputeName(ASTContext &Context) const {
 std::string SYCLUniqueStableIdExpr::ComputeName(ASTContext &Context,
                                                 const VarDecl *VD) {
   std::unique_ptr<MangleContext> Ctx{ItaniumMangleContext::create(
-      Context, Context.getDiagnostics(), SYCLMangleCallback)};
+      Context, Context.getDiagnostics(),
+      [](ASTContext &, const NamedDecl *ND) -> llvm::Optional<unsigned> {
```

Comment:
Rather than duplicating this, would it make sense to make it a local static function instead?